applog.go

```diff
@@ -0,0 +1,43 @@
+package mozlogrus
+
+import (
+	"bytes"
+	"encoding/json"
+	"os"
+	"time"
+)
+
+// AppLog implements Mozilla logging standard
+type AppLog struct {
+	Timestamp  int64
+	Time       string
+	Type       string
+	Logger     string
+	Hostname   string `json:",omitempty"`
+	EnvVersion string
+	Pid        int `json:",omitempty"`
+	Severity   int `json:",omitempty"`
+	Fields     map[string]interface{}
+}
+
+// NewAppLog returns a loggable struct
+func NewAppLog(loggerName string, logType string, msg []byte) *AppLog {
+	now := time.Now().UTC()
+	return &AppLog{
+		Timestamp:  now.UnixNano(),
+		Time:       now.Format(time.RFC3339),
+		Type:       logType,
+		Logger:     loggerName,
+		Hostname:   hostname,
```

Comment:
I believe it would be clearer to use `Hostname()` instead of the cache variable directly :)